### DISCUSSION TITLE
Clear installation instructions

### DISCUSSION THREAD

**Discussion by PetrDlouhy** [CopilotC-Nvim/CopilotChat.nvim#70] [Q&A]
I am regular (neo)vim user, but I don't very little about it's internals, add-ons and configuration. I usually reconfigure it once in few years and then I am happy with how it works.

I wanted to install CopilotChat, but I am confused about the installation process. Especially I don't know which file should I put the configuration to.
For example in the README in the Manual step is written: `3. Add to you configuration`, but I have no idea in which file. Same with Lazy.vim setup step `3. Put it in your lazy setup`.

Would it be possible to improve the README.md and put more precise installation steps with exact (or usual) locations of files?

**Comment by PetrDlouhy** [comment]
Also on Ubuntu 23.10 I got this output on the PIP installations:

```
pip install python-dotenv requests pynvim==0.5.0 prompt-toolkit
error: externally-managed-environment

× This environment is externally managed
╰─> To install Python packages system-wide, try apt install
    python3-xyz, where xyz is the package you are trying to
    install.
    
    If you wish to install a non-Debian-packaged Python package,
    create a virtual environment using python3 -m venv path/to/venv.
    Then use path/to/venv/bin/python and path/to/venv/bin/pip. Make
    sure you have python3-full installed.
    
    If you wish to install a non-Debian packaged Python application,
    it may be easiest to use pipx install xyz, which will manage a
    virtual environment for you. Make sure you have pipx installed.
    
    See /usr/share/doc/python3.11/README.venv for more information.

note: If you believe this is a mistake, please contact your Python installation or OS distribution provider. You can override this, at the risk of breaking your Python installation or OS, by passing --break-system-packages.
hint: See PEP 668 for the detailed specification.
```

I know about PIP and using environments, but I am confused in this case if I should activate some PIP virtualenv, or override this with `--break-system-packages`.

**Comment by jellydn** [comment]
Hi, @PetrDlouhy I agree with you on the installation issue. We are trying our best to make it easy for everyone to try to install. That's why I created this discussion to share how to set it up. https://github.com/CopilotC-Nvim/CopilotChat.nvim/discussions/27
